GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -428,10 +457,15 @@ private void tvGitTree_KeyDown(object sender, KeyEventArgs e)
 
         private void blameMenuItem_Click(object sender, EventArgs e)
         {
-            if (tvGitTree.SelectedNode?.Tag is GitItem gitItem)
+            if (tvGitTree.SelectedNode?.Tag is not GitItem gitItem)
             {
-                UICommands.StartFileHistoryDialog(this, gitItem.FileName, _revision, true, true);
+                return;
             }
+
+            blameToolStripMenuItem1.Checked = !blameToolStripMenuItem1.Checked;
```

Comment:
You can miss that `!` so easily here, but it is up to you.

Reply:
I guess pattern matching could be used, but I want to have `!` as our convention